serviceconnector/src/createServiceConnector/ICreateLinkerContext.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { AuthInfoBase, KnownClientType, LinkerResource } from "@azure/arm-servicelinker";
+import { IStorageAccountWizardContext } from "@microsoft/vscode-azext-azureutils";
+import { ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { TargetServiceType } from "../../constants";
+
+export interface ICreateLinkerContext extends ISubscriptionActionContext, IStorageAccountWizardContext {
+    //Source resource
+    sourceResourceUri?: string;
+    scope?: string; // This is only assigned when the source resource is a container app to indicate which container is being connected
```

Comment:
If you use a multi-line comment (`/** */` kind) then VS Code will pick it up as documentation. I think you should change this to a multi-line comment to leverage this.

<img width="594" alt="image" src="https://github.com/microsoft/vscode-azuretools/assets/12476526/d067c91a-8bc5-47b3-b98f-0dc035fbf40e">